clarifai/models/model_serving/models/inference.py

```diff
@@ -24,19 +29,22 @@ def __init__(self) -> None:
     #self.checkpoint_path: Path = os.path.join(self.base_path, "your checkpoint filename/path")
     #self.model: Callable = <load_your_model_here from checkpoint or folder>
 
-  #Add relevant model type decorator to the method below (see docs/model_types for ref.)
-  def get_predictions(self, input_data, **kwargs):
+  #@config.inference.wrap_func
```

Comment:
If you don't comment out the above, I don't think this needs to be commented out either. And then it wouldn't have to be replaced in `build_repository` as well